[Perf] add memory details in perf framework (#25724)
Azure/azure-sdk-for-js: sdk/test-utils/perf/README.md

  - making a request to the proxy server to stop playing back
- delete the live resources that we have created before

## Generating heapdumps
Launch the node process with the `--expose-gc` flag, you can then call `global.gc()` to force node to run garbage collection.

> The helper method `generateHeapDump` exported from this package can help you generate the dump file by calling `generateHeapDump(<filename>)` in the dumps folder.

Author: ```suggestion
The helper method `generateHeapDump` exported from this package can help you generate the dump file by calling `generateHeapDump(<filename>)` in the `"dumps"` folder.
```

> The helper method `generateHeapDump` exported from this package can help you generate the dump file by calling `generateHeapDump(<filename>)` in the `"dumps"` folder.

Author: ```suggestion
The helper method `generateHeapSnapshot` exported from this package can help you generate the heapsnapshot file by calling `generateHeapSnapshot(<filename>)` in the `"heapSnapshots"` folder.
```

> The helper method `generateHeapSnapshot` exported from this package can help you generate the heapsnapshot file by calling `generateHeapSnapshot(<filename>)` in the `"heapSnapshots"` folder.

Author: ```suggestion
The helper method `generateHeapSnapshot` exported from this package can help you generate the heapsnapshot file in the `"heapSnapshots"` folder by calling `generateHeapSnapshot(<filename>)`.
```
